configuration-reference/functions/funnelcount.md

```diff
@@ -0,0 +1,162 @@
+
+## Usage Examples
+
+Many datasets are time series in nature, tracking events of an entity over time.
+An example of such a dataset could be a user analytics activity log from a commerce web application. 
+
+### Example
+
+| user_id | event_time              | url                    |
+|---------|-------------------------|------------------------|
+| U1      | 2021-10-01 09:01:00.000 | /product/listing       |
+| U2      | 2021-10-01 09:17:00.000 | /product/search        |
+| U1      | 2021-10-01 09:33:00.000 | /product/details       |
+| U1      | 2021-10-01 09:47:00.000 | /cart/add              |
+| U3      | 2021-10-01 10:02:00.000 | /product/listing       |
+| U3      | 2021-10-01 10:05:00.000 | /product/search        |
+| U2      | 2021-10-01 10:06:00.000 | /product/search        |
+| U2      | 2021-10-01 10:15:00.000 | /checkout/start        |
+| U2      | 2021-10-01 10:16:00.000 | /cart/add              |
+| U3      | 2021-10-01 11:17:00.000 | /product/details       |
+| U2      | 2021-10-01 11:18:00.000 | /checkout/confirmation |
+| U3      | 2021-10-01 11:21:00.000 | /cart/add              |
+| U1      | 2021-10-01 11:33:00.000 | /cart/add              |
+| U1      | 2021-10-01 11:46:00.000 | /checkout/start        |
+| U1      | 2021-10-01 11:54:00.000 | /checkout/confirmation | 
+
+#### Funnel
+We want to analyse the following checkout funnel: 
+* */cart/add*
+* */checkout/start*
+* */checkout/confirmation*
+
+#### Counts
+We want to answer the following questions about the above funnel:
+* How many users entered the top of the funnel? 
+* How many of these users proceeded to the second step? 
+* How many users reached the bottom of the funnel after completing all steps? 
+
+#### Query
+```sql
+select 
+  FUNNEL_COUNT(
+    STEPS(
+      url = '/cart/add', 
+      url = '/checkout/start', 
+      url = '/checkout/confirmation'),
+    CORRELATED_BY(user_id)
+  ) AS counts
+from user_log 
+```
+
+| counts  |
+|---------|
+| 3, 2, 2 |
+
+> **Notes**
+> 
+> Notice that although *U1* user added to cart twice, it still counted as one conversion in the first step, as we report on unique counts rather than total events.
```

Comment:
==> user is added to the cart

Reply:
> ==> user is added to the cart

The sentence is written in the domain of the example, that of a commerce web app.
The first step in the funnel is /cart/add which represents a user adding items to their cart prior to checkout.
In the example dataset, two rows match the /cart/add url for user U1, which is interpreted as a user adding items to their cart twice.
As the aggregation function counts unique distinct values rather than total rows, the user U1 is counted as a single conversion for the first step in the funnel above.